docs/source/tools/trigger-service/index.rst

```diff
@@ -14,120 +14,139 @@ Trigger Service
-      //Minimum time interval before restarting a failed trigger. Defaults to 5 seconds.
+
+      // Minimum and maximum time interval before restarting a failed trigger. Defaults to 5 and 60 seconds respectively.
       min-restart-interval = 5s
-      //Maximum time interval between restarting a failed trigger. Defaults to 60 seconds.
       max-restart-interval = 60s
-      //Optional max HTTP entity upload size in bytes. Defaults to 4194304.
+
+      // Maximum HTTP entity upload size in bytes. Defaults to 4194304 (4 MB).
       max-http-entity-upload-size = 4194304
-      //Optional HTTP entity upload timeout. Defaults to 60 seconds.
+
+      // HTTP entity upload timeout. Defaults to 60 seconds.
       http-entity-upload-timeout = 60s
-      //Use static time or wall-clock, default is wall-clock time.
+
+      // Use static or wall-clock time. Defaults to `wall-clock`.
       time-provider-type = "wall-clock"
-      //Compiler config type to use , default or dev mode
+
+      // Compiler configuration type to use between `default` or `dev`. Defaults to `default`.
       compiler-config = "default"
-      //TTL in seconds used for commands emitted by the trigger. Defaults to 30s.
-      ttl = 60s
 
-      //Initialize database and terminate.
+      // Time-to-live used for commands emitted by the trigger. Defaults to 30 seconds.
+      ttl = 30s
+
+      // If true, initialize the database and terminate immediately. Defaults to false.
       init-db = "false"
 
-      //Do not abort if there are existing tables in the database schema. EXPERT ONLY. Defaults to false.
+      // Do not abort if there are existing tables in the database schema. EXPERT ONLY. Defaults to false.
       allow-existing-schema = "false"
 
+      // Configuration for the persistent store that will be used to keep track of running triggers across restarts.
       trigger-store {
+
+        // Mandatory. Database coordinates.
         user = "postgres"
         password = "password"
         driver = "org.postgresql.Driver"
         url = "jdbc:postgresql://localhost:5432/test?&ssl=true"
 
-        // prefix for table names to avoid collisions, empty by default
+        // Prefix for table names to avoid collisions. EXPERT ONLY. By default, this is empty and not used.
         table-prefix = "foo"
 
-        // max pool size for the database connection pool
+        // Maximum size for the database connection pool. Defaults to 12.
         pool-size = 12
-        //specifies the min idle connections for database connection pool.
+
+        // Minimum idle connections for the database connection pool. Defaults to FIXME.
         min-idle = 4
-        //specifies the idle timeout for the database connection pool.
+
+        // Idle timeout for the database connection pool. Defaults to FIXME.
         idle-timeout = 12s
-        //specifies the connection timeout for database connection pool.
+
+        // Timeout for database connection pool. Defaults to FIXME.
```

Comment:
The defaults here need to be checked and/or fixed.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13111/commits/9992ac7a1f947a40fc88278ea03d76af40439716